docs/lib/pubsub/fragments/js/getting-started.md

```diff
@@ -82,3 +82,30 @@ Amplify.addPluggable(new MqttOverWSProvider({
 ```
 
 You can integrate any MQTT Over WebSocket provider with your app. Click [here](https://docs.aws.amazon.com/iot/latest/developerguide/protocols.html#mqtt-ws) to learn more about MQTT Over WebSocket.
+
+## Removing a Provider
+
+The `removePluggable(providerName: string)` function allows you to remove a PubSub provider by supplying the name of the provider.
+
+This is useful for: working with multiple concurrent PubSub providers, reconfiguring PubSub providers when changing authentication states, and changing the IoT connection region.
```

Comment:
```suggestion
Sometimes you need to reconfigure your PubSub provider when working with multiple concurrent PubSub providers, reconfiguring authentication states, or changing the IoT connection region. To reconfigure, the PubSub provider, remove the existing provider using `removePluggable` and add an updated PubSub provider using `addPluggable`.

The `removePluggable(providerName: string)` function allows you to remove a PubSub provider by supplying the name of the provider.
```

Reply:
I'll commit this and update - would like to remove the comma after "reconfigure".